homeassistant/components/aranet/sensor.py

```diff
@@ -142,6 +143,7 @@ def _sensor_device_info_to_hass(
     if adv.readings and adv.readings.name:
         hass_device_info[ATTR_NAME] = adv.readings.name
         hass_device_info[ATTR_MANUFACTURER] = ARANET_MANUFACTURER_NAME
+        hass_device_info[ATTR_MODEL] = adv.readings.type.name
```

Comment:
The property you are using as a model here is in `IntEnum` and not a direct model name:
https://github.com/Anrijs/Aranet4-Python/blob/b927e3589cac7b23429c8ff8339c7fe9b4ca91e9/aranet4/client.py#L50-L56

```python
class AranetType(IntEnum):
    """Enum for the different Aranet devices"""
    ARANET4 = 0
    ARANET2 = 1
    ARANET_RADIATION = 2
    ARANET_RADON = 3
    UNKNOWN = 255
```

These are the names from the manufacture web:
![image](https://github.com/user-attachments/assets/10c79c5d-dc16-4a4e-bc14-08089b649ba8)

Which would not match the name from the enum.

I think the best way is to handle this upstream, by adding a `model` property which will match the manufacture name. The enum is used internally to run different code but is not a model name.